### PR TITLE
Remove unused parameter

### PR DESCRIPTION
**Pull request by sni**

### CODE REVIEW

File: src/naemon/checks_service.c [hunk 1]
```diff
@@ -466,7 +466,7 @@ int handle_async_service_check_result(service *temp_service, check_result *queue
 	else {
 
 		/* parse check output to get: (1) short output, (2) long output, (3) perf data */
-		parse_check_output(queued_check_result->output, &temp_service->plugin_output, &temp_service->long_plugin_output, &temp_service->perf_data, TRUE, FALSE);
+		parse_check_output(queued_check_result->output, &temp_service->plugin_output, &temp_service->long_plugin_output, &temp_service->perf_data, TRUE);
```
Comment (roengstrom) [inline src/naemon/checks_service.c]: Since there might exist NEB-modules that call this function, perhaps we should bump CURRENT_NEB_API_VERSION?

Comment (sni) [reply src/naemon/checks_service.c]: or should we use __attribute__ ((unused)) for now?

Comment (roengstrom) [reply src/naemon/checks_service.c]: I think that sounds like a good idea!

Comment (jacobbaungard) [reply src/naemon/checks_service.c]: Did you check to see if these were the only occurrences of that warning? I suspect that, as we compile with "-Wno-unused-parameter", there could be several places this occurs.

I am fine with using the __attribute__ annotation.

**Comment by sni** [comment]
The only reason for this PR was to support broken gcc versions on sles 11 on obs. Since there are more issues on sles 11, we just drop support for sles 11 obs builds for now.

**Pull request closed without merging**